### PR TITLE
feat(dest/fortinet): fortinet advisory 2023

### PR DESCRIPTION
**Pull request by 1uf3**

### CODE REVIEW

File: dest/fortinet/2023/FG-IR-23-069.json [hunk 1]
```diff
@@ -2,95 +2,102 @@
...
             }
           ],
-          "cvssv2": null,
           "cvssv3": {
             "base_score": 5.7,
-            "temporal_score": null,
-            "environmental_score": null,
             "vector": "CVSS:3.1/AV:L/AC:L/PR:H/UI:N/S:U/C:N/I:H/A:H/E:P/RL:X/RC:C"
           },
-          "cwe": null,
```
Comment (MaineK00n) [inline dest/fortinet/2023/FG-IR-23-069.json:84]: > A relative path traversal vulnerability [CWE-23]

File: dest/fortinet/2023/FG-IR-23-051.json [hunk 2]
```diff
@@ -2,50 +2,38 @@
...
             }
           ],
-          "cvssv2": null,
           "cvssv3": {
             "base_score": 7.2,
-            "temporal_score": null,
-            "environmental_score": null,
             "vector": "CVSS:3.1/AV:N/AC:L/PR:H/UI:N/S:U/C:H/I:H/A:H/E:X/RL:X/RC:X"
           },
-          "cwe": null,
```
Comment (MaineK00n) [inline dest/fortinet/2023/FG-IR-23-051.json:39]: > a template engine vulnerability [CWE-1336]

File: dest/fortinet/2023/FG-IR-23-050.json [hunk 3]
```diff
@@ -2,48 +2,36 @@
...
             }
           ],
-          "cvssv2": null,
           "cvssv3": {
             "base_score": 7.5,
-            "temporal_score": null,
-            "environmental_score": null,
             "vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:N/A:N/E:X/RL:X/RC:X"
           },
-          "cwe": null,
```
Comment (MaineK00n) [inline dest/fortinet/2023/FG-IR-23-050.json:39]: > An improper access control vulnerability [CWE-284]

File: dest/fortinet/2023/FG-IR-23-013.json [hunk 4]
```diff
@@ -2,78 +2,78 @@
...
             }
           ],
-          "cvssv2": null,
           "cvssv3": {
             "base_score": 5.9,
-            "temporal_score": null,
-            "environmental_score": null,
             "vector": "CVSS:3.1/AV:L/AC:L/PR:H/UI:R/S:U/C:H/I:H/A:H/E:P/RL:X/RC:R"
           },
-          "cwe": null,
```
Comment (MaineK00n) [inline dest/fortinet/2023/FG-IR-23-013.json:67]: > An improper neutralization of input during web page generation ('Cross-site Scripting') vulnerability [CWE-79]

File: dest/fortinet/2023/FG-IR-23-013.json [hunk 5]
```diff
@@ -2,78 +2,78 @@
...
-                }
-              ],
-              "hardware": [
+                  "fixed_in": "9.4.3"
+                },
+                {
+                  "description": "FortiNAC: all 9.2, fixed in 9.4.3",
+                  "cpe": "cpe:2.3:a:fortinet:fortinac:*:*:*:*:*:*:*:*",
+                  "affected": {
+                    "eq": "9.2"
```
Comment (MaineK00n) [inline dest/fortinet/2023/FG-IR-23-013.json:35]: `9.2 all versions` should be expressed as `9.2.0 through 9.2.x versions`.

**Comment by 1uf3** [comment]
not written anything regarding `Workaround for FortiOS`, `Acknowledgement` inside FG-IR-23-001.

**Review by MaineK00n — CHANGES_REQUESTED** [review]
- resolve conflicts by rebase
- describe the newly added FG-IR-23-*

**Review by MaineK00n — APPROVED** [review]

**Pull request merged**